### DISCUSSION TITLE
[New Component] [ElInputTag] 标签输入框组件

### DISCUSSION THREAD

**Discussion by codesinging** [element-plus/element-plus#14111] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `ElInputTag`

## Description

在文本输入框中输入文本，然后按空格变成标签(el-tag)形式。
标签可以删除
v-model 绑定一个数组形式的值
可以限制最多输入几个标签
可以设置标签值的类型是字符型或数值型

大致功能参考项目 https://github.com/xiispace/el-input-tag

### Additional comments

_(empty)_

**Comment by nihaoqingtuan** [comment]
现在有支持标签输入框的新组件了吗？没有的话可以用什么平替？

**Reply by l-x-f** [reply]
[PlusProComponents ](https://plus-pro-components.com/components/input-tag.html)  input-tag
